src/functionalTest/resources/features/F-012 - Create ST_CIC Staff and Judicial Role Assignments/F-012.feature

```diff
@@ -143,6 +143,90 @@ Feature: F-012 : Create Role Assignments for Special Tribunals Caseworker and Ju
     And the response has all other details as expected
     And a successful call [to delete existing role assignments corresponding to the test actorId] as in [S-130_DeleteDataForRoleAssignments].
 
+  @S-012.01
+  @FeatureToggle(DB:st_cic_wa_1_2=on) @FeatureToggle(EV:CASEWORKER_FTA_ENABLED=on)
```

Comment:
Can you set the following on all the new ST_CIC Caseworker FTAs except the pre existing S-003/a/b & S-012/a/b.
```
@FeatureToggle(DB:st_cic_wa_1_0=on)
```

As 1.2 isn't enabled in AAT at the moment: which means all these tests would be disabled on our master & nightly pipeline.

Reply:
Resolved in af871e0466eb6ca368931c518b1fc66d2ece2fb3.